### PR TITLE
Babel set up complete.

### PR DESCRIPTION
**Pull request by KingJeremyNg**
> pull latest changes
> "npm install" to install dependencies
> write tests into test.js
> "node main" to run

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+let monster2 = new monster(20, 0, 0, 5, 1, 2);
+let monster3 = new monster(15, 0, 0, 2, 4, 2);
+let monster4 = new monster(5, 0, 0, 3, 2, 3);
+let monster5 = new monster(12, 0, 0, 1, 3, 4);
+
+let turret1 = new tower(0, 0, 0, 0, "lowestHP", 2, 2, 2);
+let turret2 = new tower(0, 0, 0, 0, "highestHP", 2, 2, 2);
+let turret3 = new tower(0, 0, 0, 0, "nearest", 2, 2, 2);
+let turret4 = new tower(0, 0, 0, 0, "furthest", 2, 2, 2);
+
```
Comment (D13Lee) [inline test.js:22]: I think you are missing "first" and "last" as targeting priorities, as in closest to the end and farthest from the end

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (D13Lee) [inline package.json:1]: I'm not familiar with what this is so a brief comment at the top would be helpful

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,893 @@
+{
```
Comment (D13Lee) [inline package-lock.json:1]: A brief description at the top here would be helpful

**Review by D13Lee — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (KingJeremyNg) [reply package.json:1]: It's a config file from NPM.
Mainly for "npm install" to install all the dependencies.

File: test.js (hunk 1 above)
Comment (KingJeremyNg) [reply test.js:22]: My bad, nearest and furthest to the endpoint, not the tower.

File: package-lock.json (hunk 3 above)
Comment (KingJeremyNg) [reply package-lock.json:1]: This is automatically generated from NPM.

**Pull request merged**